### PR TITLE
backend is developed using node js

### PR DESCRIPTION
**Pull request by Nimodi**

### CODE REVIEW

File: app/controllers/order.controller.js [hunk 1]
```diff
@@ -196,63 +297,149 @@ exports.delete = (req, res) => {
...
+//   //     if(!item) {
+//   //         console.log(request.params.ItemId);
+//   //         return res.status(404).send({
+//   //             message: "Item not found with id " + req.params.itemId
+//   //         });
+//   //     }
+//   //     res.send({message: "Item deleted successfully!"});
+//   // }).catch(err => {
+//   //     if(err.kind === 'ObjectId' || err.name === 'NotFound') {
+//   //         return res.status(404).send({
```
Comment (madushankaf) [inline app/controllers/order.controller.js:405]: remove all unwanted comments

File: app/controllers/order.controller.js [hunk 2]
```diff
@@ -2,24 +2,70 @@ const Order = require("../models/order.model.js");
...
 exports.create = (req, res) => {
+  // console.log("**********");
+  // console.log(req.body.customerName);
   // Validate request
   if (!req.body.items) {
     return res.status(400).send({
       message: "Order content can not be empty"
     });
   }
-  console.log(req.body.items);
```
Comment (madushankaf) [inline app/controllers/order.controller.js:11]: please use a logger and get rid of console logs

File: app/controllers/order.controller.js (hunk 1 above)
Comment (Nimodi) [reply app/controllers/order.controller.js:405]: Got it

File: app/controllers/order.controller.js (hunk 2 above)
Comment (Nimodi) [reply app/controllers/order.controller.js:11]: Got it

**Pull request merged**